examples/quickstart/service.py

```diff
@@ -8,6 +8,9 @@
 svc = bentoml.Service("iris_classifier", runners=[iris_clf_runner])
 
 
-@svc.api(input=NumpyNdarray(), output=NumpyNdarray())
+@svc.api(
+    input=NumpyNdarray.from_sample(np.array([[4.9, 3.0, 1.4, 0.2]], dtype=np.double)),
```

Comment:
This helps with showing an example in the swagger page:
<img width="555" alt="Screen Shot 2022-08-19 at 3 11 05 PM" src="https://user-images.githubusercontent.com/489344/185713734-60721f34-b740-4439-a703-9390a46369a6.png">